crates/pixi_build_types/src/lib.rs

```diff
@@ -24,7 +24,7 @@ use serde::{Deserialize, Serialize};
 /// The constraint for the pixi build api version package
 /// Adding this constraint when solving a pixi build backend environment ensures that
 /// a backend is selected that uses the same interface version as Pixi does
-pub const PIXI_BUILD_API_VERSION_NAME: LazyLock<PackageName> =
+pub static PIXI_BUILD_API_VERSION_NAME: LazyLock<PackageName> =
```

Comment:
Important one! clippy was complaining about interior mutability and suggested to use static here. I checked the LazyLock source code - it make sense to have static here